docs/enterprise-guide/full-app-embedding.md

```diff
@@ -63,6 +64,57 @@ The main elements you'll need to embed Metabase in your app are:
 
   { "metabase": { "type": "location", "location": LOCATION_OBJECT_OR_URL }}
 
+## Overriding selected full-app components with parameters
+
+You can add query parameters to the URL to toggle various full-app features.
+
+E.g., the URL
+
+```
+http://localhost:3000/?top-nav=false&side_nav=false
+```
+
+would disable the top and side nav:
+
+![Top nav and side nav disabled](./images/full-app-embedding/no-top-no-side.png)
+
+### `top_nav`
+
+Entire top navigation bar, with optional search and new button.
+
+![Top nav bar](./images/full-app-embedding/top-nav.png)
+
+The top bar is hidden by default. Additionally, if you enable the top bar (`top_bar=true`), you can turn on other hidden-by-default top bar options:
+
+- `search`: Search bar within the top nav.
+- `new_button`: “New” CTA that lets users create questions and more.
+
+### `side_nav`
+
+The main navigation bar.
+
+![Side nav](./images/full-app-embedding/side-nav.png)
+
+The navigation sidebar is hidden by default, aside from `/collection` and home page product routes. If you want people to be able to minimize the sidebar, you MUST enable the `top-nav`.
+
+### `header`
+
+The header, visible by default, only applies to dashboards and questions. The header refers to the info and buttons above a question or dashboard. The header includes the title, additional info, and the action buttons (Filter and Summarize).
+
+If you enable the header, these sub-components are visible by default:
+
+#### `additional_info`
+
+Applicable to dashboards and questions. Refers to the gray text “Edited X days ago by FirstName LastName”, as well as the collection, database, and table information.
```

Comment:
If `additional_info` is toggled off, does that also remove the option to view a question's history / revert it?

Asking because I usually click on the gray text to get to the history (but maybe there's other paths I don't know about).

Reply:
No. You can also view the history via the info button.